doc/Makefile

```diff
@@ -30,11 +31,15 @@ all: aspect.tag manual.pdf
 	@perl manual/cookbooks/annotate.pl $< >$@
 	@for i in `seq 1 10`; do sed -i 's/{\([^!]*\)!\([^!]*\)!\([^!]*\)!\([^}]*\)}/{\1!\2!\3\/\4}/' $@; done
 
+# Generate changes.h from the bits in modules/changes/
+modules/changes.h: modules/changes/* modules/create_changes.sh modules/current_changes_*
```

Comment:
Looking through this again, this won't work -- `make` does not expand wildcards. You'll have to write `$(shell echo modules/changes/* modules/create_changes.sh modules/current_changes_*)`